app/packages/core/src/components/Filters/CategoricalFilter.tsx

```diff
@@ -102,18 +102,25 @@ const Wrapper = ({
   const selectedSet = new Set(selected);
   const setExcluded = excludeAtom ? useSetRecoilState(excludeAtom) : null;
   const sorting = useRecoilValue(fos.sortFilterResults(modal));
-  const counts = Object.fromEntries(results);
+  const filteredResults = schema?.ftype.includes("ListField")
```

Comment:
I do think this is something that should happen in [`noneCount`](https://github.com/voxel51/fiftyone/blob/a40c2dd272b58b24d54760e84847ca1f508301ac/app/packages/state/src/recoil/aggregations.ts#L84) because this behavior should be shared by any other components. Defining in just this component could lead to other inconsistencies